### PR TITLE
Add my avatar and name

### PR DESCRIPTION
**Pull request by Yash-Garg**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [ ] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://yash-garg.github.io/fossasia-gci2018-org/)

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -156,3 +156,8 @@
 - name: Navan Chauhan
   github: navanchauhan
   image: n3v0n.png
+
+- name: Yash Garg
+  github: Yash-Garg
+  image: yashgarg.png
+  twitter: yashgarg1803
```
Comment (jogendra) [inline _data/students.yml]: Remove this.

Comment (Yash-Garg) [reply _data/students.yml]: Okay

**Comment by Yash-Garg** [comment]
I will force push this in 5 mins

**Comment by Yash-Garg** [comment]
Ohh okay

**Comment by Yash-Garg** [comment]
@jogendra Done!

**Comment by sarnava1** [comment]
@Yash-Garg please now resubmit the task as your PR has been successfully merged. Thank you.

**Comment by Yash-Garg** [comment]
okay

**Pull request merged**